internal/backend/runtime/omni/controllers/omni/internal/task/machine/poll.go

```diff
@@ -314,6 +314,7 @@ func pollTalosMachineStatus(ctx context.Context, c *client.Client, info *Info) e
 		}
 
 		return &specs.MachineStatusSpec_TalosMachineStatus{
+			Version:   r.Metadata().Version().Value(),
 			UpdatedAt: timestamppb.New(r.Metadata().Updated()),
```

Comment:
so this is Talos timestamp (I guess?)

Reply:
yeah, I've messed it up. Sorry I'm still fighting with my network setup... Will submit the fix. Now for sure.